### PR TITLE
[NMS] Add Gateway create and edit

### PR DESCRIPTION
**Pull request by HannaFar**
## Summary

Add Gateway create and edit dialog
https://www.figma.com/file/hw8iheccicYs6tW7gsFsPK/Magma-Client-Facing-Final-Design?node-id=372%3A4194

create

### CODE REVIEW

**Comment by karthiksubraveti** [comment]
One thing i realized when making subscription related provisioning pages is that sometimes we have to propagate the state all the way upto Overview page. For e.g. We currently have this bug in enodeb provisioning where detailed pages get updated but top overview page doesn't get updated.
So in subscriber provisioning i ended using context for holding state and using set methods to propagate state up all the way to the top. In future, we could clean this up with better state management methods i believe.

out

File: nms/app/fbcnms-projects/magmalte/app/views/equipment/GatewayDetailConfigEdit.js [hunk 1]
```diff
@@ -0,0 +1,758 @@
...
+    fontWeight: typography.button.fontWeight,
+    fontSize: typography.button.fontSize,
+    lineHeight: typography.button.lineHeight,
+    letterSpacing: typography.button.letterSpacing,
+
+    '&:hover': {
+      background: colors.primary.mirage,
+    },
+  },
+  input: {
```
Comment (karthiksubraveti) [inline nms/app/fbcnms-projects/magmalte/app/views/equipment/GatewayDetailConfigEdit.js]: @ekfuhrmann is making changes to global dialog settings, might be useful to make those associated changes here.

File: nms/app/fbcnms-projects/magmalte/app/views/equipment/EquipmentDashboard.js [hunk 2]
```diff
@@ -235,6 +236,9 @@ function EquipmentDashboardInternal({
                 </Button>
               </Grid>
               <Grid item>
+                {tabPos == 0 && (
```
Comment (karthiksubraveti) [inline nms/app/fbcnms-projects/magmalte/app/views/equipment/EquipmentDashboard.js:244]: both seem to have same title ?

**Review by karthiksubraveti — APPROVED** [review]
Changes LGTM. If possible please post a screen recording of the changes. You can convert mov to gif and attach it to the review. Thanks

**Comment by 119Vik** [comment]
Can one of the admins verify this patch?

**Comment by karthiksubraveti** [comment]
@HannaFar the copyright information recently changed and that's causing eslint failure. Can you update this. We can merge the change after this

**Pull request merged**